pkgs/applications/misc/makeself/default.nix

```diff
@@ -12,6 +12,7 @@ stdenv.mkDerivation rec {
   };
 
   patchPhase = ''
+    sed -e "s|/bin/rm|rm|" -i makeself-header.sh
```

Comment:
Can you make this a patch instead and put it in `patches = []`, so we can remove it when we update the package?
`patchPhase` must be also renamed to `postPatch` otherwise patches are not applied.

Reply:
If the patch from the commit applies you can also try `fetchpatch` on the commit `7a465c910bb0b7456866c09d129be388361158ef`

Reply:
Thanks for the review, and sorry for the late response. I've rebased this PR and incorporated your suggestions.